addon/utils/serializer-partial-model-handler.js

```diff
@@ -0,0 +1,33 @@
+export default {
+  preNormalize: function(modelClass, resourceHash, context, normalizePartialRelationshipStrategy) {
+    if (modelClass._isPartialModel) {
+      partialDescriptors(modelClass).forEach(descriptor => {
+        normalizePartialRelationshipStrategy(modelClass, resourceHash, descriptor, context);
```

Comment:
You need to `.call` `normalizePartialRelationshipStrategy` otherwise if someone is accessing `this` in their callback, it will explode